pkgs/applications/version-management/gogs/deps.nix

```diff
@@ -0,0 +1,452 @@
+      sha256 = "0f979lil952gy7gr61xwdj0nid2wkxjd7gzzr9iki4rrq67h23vy";
+    };
+  }
+  {
+    goPackagePath = "github.com/gogits/chardet";
+    fetch = {
+      type = "git";
+      url = "https://github.com/gogits/chardet";
+      rev = "2404f777256163ea3eadb273dada5dcb037993c0";
+      sha256 = "1dki2pqhnzcmzlqrq4d4jwknnjxm82xqnmizjjdblb6h98ans1cd";
+    };
+  }
+  {
+    goPackagePath = "github.com/gogits/cron";
+    fetch = {
+      type = "git";
+      url = "https://github.com/gogits/cron";
+      rev = "7f3990acf1833faa5ebd0e86f0a4c72a4b5eba3c";
+      sha256 = "0y7z62g727pygnm5y1jp4i8izz4kzrnih8ihapxhin0isbfd3b6y";
+    };
+  }
+  {
+    goPackagePath = "github.com/gogits/git-module";
+    fetch = {
+      type = "git";
+      url = "https://github.com/gogits/git-module";
+      rev = "b3009dc4f5842cf9e2e80fef1e125e79c38e4949";
+      sha256 = "1f5mms15hknnj17nvb5dwwk8fcm8a9msy9qdjvkplihxgaqm0amz";
+    };
+  }
+  {
+    goPackagePath = "github.com/gogits/go-gogs-client";
+    fetch = {
+      type = "git";
+      url = "https://github.com/gogits/go-gogs-client";
+      rev = "d8aff570fa22d4e38954c753ec8b21862239b31d";
+      sha256 = "07gmvvljd48jwpkgsqlcp19rphn7ybcd4hzf7zahhl4dd4b8p4p3";
+    };
+  }
+  {
+    goPackagePath = "github.com/issue9/identicon";
+    fetch = {
+      type = "git";
+      url = "https://github.com/issue9/identicon";
+      rev = "d36b54562f4cf70c83653e13dc95c220c79ef521";
+      sha256 = "0y82b3gq8rpqglvf3lsqhgp5djfdammwd1w24k3i97iqls0rch7l";
+    };
+  }
+  {
+    goPackagePath = "github.com/jaytaylor/html2text";
+    fetch = {
+      type = "git";
+      url = "https://github.com/jaytaylor/html2text";
+      rev = "8fb95d837f7d6db1913fecfd7bcc5333e6499596";
+      sha256 = "0pzyidlp5jmzl5smhr9f8a81qpfls80j5m7156b032fvyvjwanlr";
+    };
+  }
+  {
+    goPackagePath = "github.com/klauspost/compress";
+    fetch = {
+      type = "git";
+      url = "https://github.com/klauspost/compress";
+      rev = "d0763f13d86e630f5d3ea9fa848a6ecc68255297";
+      sha256 = "18nf5iqsrz2xq899zy9gvi1s0gaxn4zwjgkxxvlqmjzs7waq946y";
+    };
+  }
+  {
+    goPackagePath = "github.com/klauspost/cpuid";
+    fetch = {
+      type = "git";
+      url = "https://github.com/klauspost/cpuid";
+      rev = "09cded8978dc9e80714c4d85b0322337b0a1e5e0";
+      sha256 = "05l8pfch0gvxh0khapwxhsk4xajn40vbjr360n49vh2z5531v2xq";
+    };
+  }
+  {
+    goPackagePath = "github.com/klauspost/crc32";
+    fetch = {
+      type = "git";
+      url = "https://github.com/klauspost/crc32";
+      rev = "19b0b332c9e4516a6370a0456e6182c3b5036720";
+      sha256 = "0fcnsf1m0bzplgp28dz8skza6l7rc65s180x85rzbdl9l3zzi43r";
+    };
+  }
+  {
+    goPackagePath = "github.com/lib/pq";
+    fetch = {
+      type = "git";
+      url = "https://github.com/lib/pq";
+      rev = "50761b0867bd1d9d069276790bcd4a3bccf2324a";
+      sha256 = "1bz46slzri958wdnvlvyx12bp3pmy8iny5m0m16i3n8h1q84dhp5";
+    };
+  }
+  {
+    goPackagePath = "github.com/mattn/go-sqlite3";
```

Comment:
It works fine without.
Other packages ([1](https://github.com/NixOS/nixpkgs/blob/98a9d815e05dd56aba0f9040d96092335cd90444/pkgs/shells/elvish/default.nix), [2](https://github.com/NixOS/nixpkgs/blob/98a9d815e05dd56aba0f9040d96092335cd90444/pkgs/development/tools/textql/default.nix)) that depend on `go-sqlite3` also don't include it.